src/App.js

```diff
@@ -10,6 +10,9 @@ import { Provider as PaperProvider, DefaultTheme } from 'react-native-paper';
 import Navigation from 'Navigation';
 
 import 'config/corejs';
+import ignoreLogs from 'config/ignoreLogs';
+
+ignoreLogs(['EventEmitter.addListener', 'Require cycle'], 'error');
```

Comment:
I'm ok to ignore `'EventEmitter.addListener'`, but `'Require cycle'` is important. We should add an issue to fix the require circle. I hope that with the SDK 2.0 we will get rid of Require circles.
https://stackoverflow.com/a/10872944